tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -326,13 +326,14 @@ Check that superseded warning link takes user to superseding-publication data-ca
 
     user clicks element    testid:superseded-by-link
 
-    user waits until page contains    Browse our open data    %{WAIT_SMALL}
+    user waits until h1 is visible    Browse our open data
 
     user waits until page contains    ${PUBLICATION_NAME_SUPERSEDE}    %{WAIT_SMALL}
+    user checks page contains radio    ${RELEASE_NAME_SUPERSEDE}
```

Comment:
I added this check in for the radio button `Financial Year 2000-01` because up until this point the previous checks will still pass if clicking the link doesn't navigate anywhere.

`Financial Year 2000-01` should only be on the page if we've successfully navigated to the superseding publication.